docs/core/deploying/trim-self-contained.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Trim self-contained deployments and executables
+description: Learn how to trim self-contained deployments and executables to reduce their size. 
+author: jamshedd
+ms.author: jamshedd
+ms.date: 01/23/2020
+---
+# Trim self-contained deployments and executables
+
+When publishing self-contained deployments (SCD) or self-contained executables (SCE), the .NET Core runtime is bundled together with the application. This bundling adds a significant amount of content to your packaged application.
```

Comment:
I think we should drop the executable part, maybe @richlander or @KathleenDollard can clarify. self-contained deployments end up giving you an executable so this seems redundant.

Reply:
I agree unless you also want to reference single exe creation. Otherwise, the article LGTM.

Reply:
I thought SCD only referred to the "legacy" behavior that didn't include the exes.